app.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controller");
+const endpoints = require("./endpoints.json");
+const { getArticleById } = require("./controllers/articles.controller");
+
+const app = express();
+
+app.get("/api", (req, res) => {
+  res.json(endpoints);
+});
+
+app.get("/api/topics", getAllTopics);
+
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad request, invalid id/not a number" });
```

Comment:
This might be tricky later if this error is caused by something other than an ID, just something to be aware of